### ISSUE TITLE
Bug: `unicorn/import-style` incorrectly reports type-only imports from chalk

### ISSUE THREAD

**Issue by NishargShah** [sindresorhus/eslint-plugin-unicorn#3598]
## Description

The `unicorn/import-style` rule incorrectly reports an error for type-only imports from `chalk`.

### Code

```ts
import type { ColorName } from 'chalk';
```

### Error

```text
Use default import for module `chalk`. eslint(unicorn/import-style)
```

## Expected Behavior

The rule should ignore `import type` declarations. Since they are erased at compile time and are used solely for TypeScript type checking, they should not be subject to default import style enforcement.

The following code is perfectly valid and should not produce any lint errors:

```ts
import type { ColorName } from 'chalk';
```

## Actual Behavior

`unicorn/import-style` reports:

```text
Use default import for module `chalk`.
```

even though the import is type-only and does not import any runtime value.
